### DISCUSSION TITLE
For what is cache used?

### DISCUSSION THREAD

**Discussion by Fabizocker456** [termux/termux-app#3499] [Q&A] [answered]
Of the 16G of internal storage that Termux used on my phone after about a year of frequent usage, about 8G were "cache", according to the settings app. Clearing had no noticeable effects, except freeing a lot of storage space.

What is the cache for, are there any negative consequences to clearing it, and is there any way I can prevent the cache growing to such sizes in the future?

**Comment by agnostic-apollo** [comment, marked as answer]
Cache would mainly be used for cached deb files for `apt`. Can also use `apt`/`pkg` `clean` command. It is safe to clean.
